dashboard-ui/src/lib/color.tsx

```diff
@@ -0,0 +1,103 @@
+// Copyright 2024-2025 Andres Morey
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useEffect, useState } from 'react';
+
+const LIGHT_THEME_COLORS = [
+  '#d63031', // Strong Red
+  '#00b894', // Strong Green
+  '#0984e3', // Strong Blue
+  '#fdcb6e', // Strong Yellow
+  '#e17055', // Strong Orange
+  '#6c5ce7', // Strong Purple
+  '#00cec9', // Strong Cyan
+  '#e84393', // Strong Magenta
+  '#8b4513', // Strong Brown
+  '#e91e63', // Strong Pink
+  '#006064', // Strong Teal
+  '#9c27b0', // Strong Lavender
+  '#b71c1c', // Strong Maroon
+  '#827717', // Strong Olive
+  '#1a237e', // Strong Navy
+  '#689f38', // Strong Lime
+  '#ff8f00', // Strong Amber
+  '#388e3c', // Strong Mint
+  '#ff5722', // Strong Deep Orange
+  '#424242', // Strong Grey
+];
+
+const DARK_THEME_COLORS = [
+  '#ff5757', // Bright Red
+  '#5cb85c', // Bright Green
+  '#4da6ff', // Sky Blue
+  '#ffeb3b', // Bright Yellow
+  '#ff9800', // Bright Orange
+  '#9c5cff', // Bright Purple
+  '#26d0ce', // Bright Cyan
+  '#ff4081', // Bright Pink
+  '#d4a574', // Light Brown
+  '#f06292', // Light Rose
+  '#4db6ac', // Light Teal
+  '#ba68c8', // Light Lavender
+  '#ef5350', // Light Coral
+  '#cddc39', // Light Lime
+  '#7986cb', // Light Indigo
+  '#8bc34a', // Fresh Lime
+  '#ffc107', // Amber Gold
+  '#81c784', // Light Mint
+  '#ff7043', // Light Deep Orange
+  '#dddbd9', // Light White
+];
+
+export type ContainerColorProps = {
+  containerKey: string;
+  size?: 'small' | 'medium';
+  className?: string;
+  theme?: 'light' | 'dark';
+};
+
+export const ContainerColor = ({ containerKey, size = 'medium', className = '', theme = 'light' }: ContainerColorProps) => {
```

Comment:
You can use the `useTheme()` hook here instead of passing `theme` as an argument.